### DISCUSSION TITLE
Pimping up a board card with image/markdown and link

### DISCUSSION THREAD

**Discussion by bpannier** [marcusolsson/obsidian-projects#900] [Show and Tell]
Hi, 

I would like to share my outcome of how to pimp up a card to make it look "nicer". I share it here as it was not too easy to include an image, keep a link to the note and change the link size.

The result:
<img width="239" alt="Screenshot 2024-06-07 at 10 02 16" src="https://github.com/marcusolsson/obsidian-projects/assets/7093174/343e2c52-71b9-4e86-b61d-82ae3c4d4c49">

How to:
- Have a property named "title" and in the "Configure field" dialog (in a table view), enable the "Enable rich text formatting"
- In the "Board setting" dialog assign "Customize card header" to "title"
- I have a summary property to show, which is straight forward ("Include fields")
- The hard part was to find the right Markdown within the property, the example is:

`title: "![foo|200](linked-brain-network.jpg) <font size=+2>[Obsidian](<Obsidian & PKM setup process>)</font>"`

Hope this helps some people.

Ben

**Comment by Janita-S** [comment]
Thanks for this tip.  It helps!  😀
